.github/workflows/power9.yml

```diff
@@ -0,0 +1,118 @@
+name: github-POWER9
+
+# Only allow manual runs until at2 runners are available.
+on: workflow_dispatch
+  #pull_request:
+  #  paths-ignore:
+  #  - '**/*.rst'
+  #  - '**/*.md'
+  #  - '**/requirements.txt'
+  #  - '**/*.py'
+  #  - 'docs/**'
+  #  types: [ opened, reopened, synchronize ]
+
+permissions:
+  contents: none
+
+# Cancels any in progress 'workflow' associated with this PR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  PR_POWER9_GCC930_OPENMP_SERIAL_LEFT_REL:
+    name: PR_POWER9_GCC930_OPENMP_SERIAL_LEFT_REL
+    runs-on: [kk-env-gcc-9.3.0-latest]
+
+    strategy:
+      matrix:
+        include:
+          - backend: "SERIAL"
+            serial: "ON"
+            openmp: "OFF"
+          - backend: "OPENMP"
+            serial: "OFF"
+            openmp: "ON"
+          - backend: "SERIAL_OPENMP"
+            serial: "ON"
+            openmp: "ON"
+    
+    steps:
+      - name: checkout_kokkos_kernels
+        uses: actions/checkout@v3
+        with:
+          path: kokkos-kernels
+
+      - name: checkout_kokkos
+        uses: actions/checkout@v3
+        with:
+          repository: kokkos/kokkos
+          ref: ${{ github.base_ref }}
+          path: kokkos
+
+      - name: configure_kokkos
+        run: |
+          mkdir -p kokkos/{build-${{ matrix.backend }},install-${{ matrix.backend }}}
+          cd kokkos/build-${{ matrix.backend }}
+          GPP=$(which g++)
+          cmake -DCMAKE_CXX_COMPILER=$GPP \
+            -DCMAKE_CXX_FLAGS="-O3 -Wall -Wunused-parameter -Wshadow -pedantic -Werror -Wsign-compare -Wtype-limits -Wignored-qualifiers -Wempty-body -Wclobbered -Wuninitialized" \
+            -DCMAKE_EXE_LINKER_FLAGS= \
+            -DCMAKE_INSTALL_PREFIX=$PWD/../install-${{ matrix.backend }} \
+            -DKokkos_ENABLE_SERIAL=${{ matrix.serial }} \
+            -DKokkos_ENABLE_OPENMP=${{ matrix.openmp }} \
+            -DKokkos_ARCH_PASCAL60=ON \
+            -DKokkos_ARCH_POWER8=ON \
```

Comment:
Is it Power8 or Power9? because the name of that workflow is power9... we should check eventually...

Reply:
Same as above WRT cm_test_all_sandia.

Reply:
@e10harvey what machine was this tested on? Pascal60 and Power8 have not been options available in cm_test_all_sandia since white/ride were decommissioned

Reply:
Also this is a Serial / OpenMP build so I the GPU Arch does not need to be specified, we can just drop it.

Reply:
This has not been tested yet, only the 7 items checked off in the list above have been tested and confirmed to pass.

Reply:
Also, FWIW, this is the exact CMake command being generated by cm_test_all_sandia for `KokkosKernels_PullRequest_GCC930_Light_Tpls_GCC930_Tpls_CLANG13CUDA10`.

Reply:
So same thing as what I wrote below, can we remove it from the PR and only add it later when it is ready?

Reply:
Sure, no problem.